pkg/record/interface.go

```diff
@@ -30,3 +37,79 @@ type JSONMarshaller struct {
 func (m JSONMarshaller) Marshal(_ context.Context) ([]byte, error) {
 	return json.Marshal(m.Object)
 }
+
+// ErrSkipRecord instructs Collect to ignore the provided recorder.
+var ErrSkipRecord = fmt.Errorf("skip recording")
```

Comment:
errors.New

Reply:
NOOOO.  I hate errors.New and overlaps with the api errors package.

Reply:
I'm going to fix this by removing the need for it.